package.json

```diff
@@ -18,6 +18,9 @@
     "*.css": "stylelint"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.18",
```

Comment:
fortawesome?

Reply:
Its really Font Awesome haha, not sure why thats their install.  But I just followed these instructions and it was pretty easy: 
https://fontawesome.com/how-to-use/on-the-web/using-with/react